index.html

```diff
@@ -19,26 +19,43 @@
 	<script src="https://js.arcgis.com/4.15/"></script>
 
 	<script>
-		var point_template_tree = {
-			title: "{Name}",
-			content: "Cây này ở <b>{Location}</b>."
-		};
-		var point_template_light = {
-			title: "{Name}",
-			content: "Bóng đèn này ở <b>{Location}</b>."
-		};
-		var point_template_chair = {
+		// var point_template_tree = {
+		// 	title: "{Name}",
+		// 	content: "Cây này ở <b>{Location}</b>."
+		// };
+		// var point_template_light = {
+		// 	title: "{Name}",
+		// 	content: "Bóng đèn này ở <b>{Location}</b>."
+		// };
+		// var point_template_chair = {
+		// 	title: "{Name}",
+		// 	content: "Cái ghế này ở <b>{Location}</b>."
+		// };
+		// var point_template_area = {
+		// 	title: "{Name}",
+		// 	content: "Khu vực này ở <b>{Location}</b>."
+		// };
+		// var point_template_university = {
+		// 	title: "{Name}",
+		// 	content: "Trường ở <b>{Location}</b>."
+		// };
+
+		///Phường 4
+		var point_template_P4_hospital = {
+					title: "{Name}",
+					content: "<b>{Location}</b>"
+				};
+		var point_template_P4= {
 			title: "{Name}",
-			content: "Cái ghế này ở <b>{Location}</b>."
+			content: "<b>{Location}</b>."
 		};
+
+		////////////////
 		var point_template_area = {
-			title: "{Name}",
-			content: "Khu vực này ở <b>{Location}</b>."
-		};
-		var point_template_university = {
-			title: "{Name}",
-			content: "Trường ở <b>{Location}</b>."
-		};
+						title: "{Name}",
+						content: "<b>{Location}</b>."
+					};
+

```

Comment:
I think `point_template_area` is not in your scope. If i was wrong please ignore this comment.

Reply:
point_template_area -> i didn't edit it. it still the same

Reply:
<img width="696" alt="Screenshot 2024-11-22 at 11 13 19 AM" src="https://github.com/user-attachments/assets/72ba7851-e8d2-4aa9-a964-81e205286edd">
chỗ này em nhìn vào cái file của em, chỗ đoạn xanh em bấm vô, nó sẽ xổ ra cái menu, em chọn Revert Change là được @wsoft416

Reply:
Nhung đã lấy lại file và copy điều chỉnh lại như cũ chỉ khác là var đổi thành let